### ISSUE TITLE
Exclude Excalidraw directory from search results

### ISSUE THREAD

**Issue by 411A** [zsviczian/obsidian-excalidraw-plugin#665]
When I search for a timestamp, I get so many results from excalidraw folder, I need an option to exclude the whole directory from my Obsidian file explorer and search results (only accessible when searching from other notes).

**Comment by zsviczian** [comment]
I don't think you really want to exclude drawings from search, as the ability to find images based on their content is a very useful feature.

I have two options for you:

1) Obsidian has a feature to exclude folders, so if this is what you really want to do, it's easy. Use this feature:
![image](https://user-images.githubusercontent.com/14358394/175073377-fc58b931-0bfd-400d-9c80-337199bb1c6a.png)

As an alternative, and I recommend this option, you can configure Excalidraw to save the drawing in a compressed format. If you do this, you will only get Excalidraw results in search if a text element or the filename of an attached image matches your search term. 

![image](https://user-images.githubusercontent.com/14358394/175074295-5bb80238-32b5-48f2-a93e-63ff835c61fd.png)
